src/SFA.DAS.AdminService.Web.Tests/Services/Gateway/OrganisationChecks/Orchestrator/OrganisationRiskTests.cs

```diff
@@ -62,7 +62,7 @@ public void Check_organisation_risk_details_are_returned(string organisationType
 
             var viewModel = response.Result;
 
-            Assert.AreEqual(UKRLPLegalName, viewModel.UkrlpLegalName);
+            Assert.AreEqual(UKRLPLegalName, viewModel.ApplyLegalName);
```

Comment:
As mentioned elsewhere regarding name check

Reply:
We need to use only ApplyLegalName as common property in RoatpGatewayPageViewModel. And it is the only one populated by PopulatePageCommonDetails.
If someone needs UkrlpLegalName in their particular ViewModel, they can deal with it in their implementation.